src/Nethermind/Nethermind.Merge.Plugin/Handlers/EngineRpcCapabilitiesProvider.cs

```diff
@@ -52,6 +52,12 @@ public EngineRpcCapabilitiesProvider(ISpecProvider specProvider)
             _capabilities[nameof(IEngineRpcModule.engine_getPayloadV4)] = (spec.RequestsEnabled, spec.RequestsEnabled);
             _capabilities[nameof(IEngineRpcModule.engine_newPayloadV4)] = (spec.RequestsEnabled, spec.RequestsEnabled);
             #endregion
+
+            #region Osaka
+            _capabilities[nameof(IEngineRpcModule.engine_getInclusionList)] = (spec.IsEip7805Enabled, spec.IsEip7805Enabled);
```

Comment:
should `engine_getInclusionList` be named `engine_getInclusionListV1`? More of a spec question